src/AI.java

```diff
@@ -211,11 +211,11 @@ static int bestColumn(Board board, int difficulty) {
 		if (difficulty == 0) {
 			return (int) (Math.random() * 7); // it's incredibly easy
 		} else if (difficulty == 1) { // modify depth based on current difficulty
-			depth = 3;
+			depth = 2;
 		} else if (difficulty == 2) {
-			depth = 4;
+			depth = 3;
 		} else {
-			depth = 8;
+			depth = 4;
```

Comment:
I think it would be better to update this in #43 so we don't have any merge conflicts later